introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-fulltraining-highlevel.ipynb

```diff
@@ -7,11 +7,14 @@
     "# End-to-End Multiclass Image Classification Example with SageMaker SDK and SageMaker Neo\n",
```

Comment:
<p>Have you forgot the section where you create the deploy_amt_model variable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes! Fixed.